### ISSUE TITLE
样式更新

### ISSUE THREAD

**Issue by 0x-jerry** [cwxyz007/hexo-theme-simple#9]
这样更容易区分每一篇文章

![image](https://user-images.githubusercontent.com/14226737/61281265-f15bf700-a7eb-11e9-9acb-5cbb6d51a069.png)

**Comment by 0x-jerry** [comment]
```css
.body {
    margin: 0 auto;
    background-color: #f5f5f5;
}
.simple-index .article-excerpt:hover {
    box-shadow: 0px 4px 15px 0px rgba(255, 0, 0, 0.22);
    transform: translateY(-4px);
}
.simple-index .article-excerpt {
    box-shadow: 0 0 0px #ffffff;
    transition: all 0.2s ease;
    position: relative;
    display: block;
    padding: 20px 50px;
    margin-bottom: 30px;
    overflow: hidden;
    cursor: pointer;
    background: #fff;
}
```

**Comment by 0x-jerry** [comment]
去除阅读全文，标签一直显示

![image](https://user-images.githubusercontent.com/14226737/61282225-8c090580-a7ed-11e9-9ae7-ff4e10548998.png)
![image](https://user-images.githubusercontent.com/14226737/61282295-ab079780-a7ed-11e9-9223-fb8094d30250.png)

![image](https://user-images.githubusercontent.com/14226737/61282283-a642e380-a7ed-11e9-95e2-d981cc0c374c.png)

**Comment by 0x-jerry** [comment]
Closed 7f6f1895284f8e3a8d880c32b586f52ff6edd9d7
